cypress-tests/cypress/support/commands/general.ts

```diff
@@ -43,6 +76,7 @@ export function getRandomString(length = 8) {
 export default function registerGeneralCommands() {
   Cypress.Commands.add("getIframe", getIframe);
   Cypress.Commands.add("getDataCy", getDataCy);
+  Cypress.Commands.add("getUserData", getUserData);
```

Comment:
Do not add this as command, a utility function works perfectly well.

See: https://github.com/SwissDataScienceCenter/renku/pull/3781#discussion_r1781173793.